api/registrations/serializers.py

```diff
@@ -556,6 +561,11 @@ def get_embargo_end_date_by_version(self, validated_data):
         New API versions should pass in "embargo_end_date"
         """
         if self.expect_cleaner_attributes(self.context['request']):
+            if validated_data.get('lift_embargo'):
+                raise exceptions.ValidationError(
+                    f'The field lift_embargo was '
```

Comment:
Ideally you'd use the `pointer` key to specify which field was incorrect, and then the message would be "Deprecated in {Version}. Use emargo_end_date instead." Check out `api.base.exceptions`, there are a couple of error formatters that you could consider using (one is even specific to Validation errors).